src/Inquiry.js

```diff
@@ -0,0 +1,75 @@
+class Inquiry {
```

Comment:
I love the idea of a class that focuses on getting info in response to questions.